client/src/components/annotators/ImageAnnotator.vue

```diff
@@ -12,8 +12,23 @@ export default {
       required: true,
     },
   },
-
+  data() {
+    return {
+      loadingVideo: false,
+    };
+  },
   created() {
+    this.playCache = 1; // seconds required to remain playing (should be less that cache seconds)
+    this.cacheSeconds = 3; // seconds to cache
+    this.frontBackRatio = 0.90; // 90% forward frames, 10% backward frames when caching
+    // playCache needs to be less than the adjusted cache seconds to work properly
```

Comment:
I don't understand this comment.

Reply:
There is the standard cache, which is how many seconds of images it should download represented by `cacheSeconds`
There is the `playCache` this cache looks forward X seconds from the current frame and sees if the frames are all loaded.  If not it will display a Loading spinner.
The `frontBackRatio` resolves the part of the issue of backwards seeking.  If you jump forward using a the mouse it will dedicate a certain portion to caching data behind the current frame.